pebbles/views/namespaced_keyvalues.py

```diff
@@ -111,11 +111,13 @@ def put(self, namespace, key):
         if not namespaced_keyvalue:
             logging.warn("no NamespacedKeyValue object found for namespace %s with key %s" % (namespace, key))
             abort(404)
+
         # Check for concurrency
-        namespaced_keyvalue_updated = namespaced_keyvalue_query.filter_by(updated_ts=updated_version_ts).first()
-        if not namespaced_keyvalue_updated:
-            logging.warn("trying to modify an outdated record")
-            return {'error': 'CONCURRENT_MODIFICATION_EXCEPTION'}, 409
+        if key != 'backend_config':
```

Comment:
This means that the backend_config record won't ever check for concurrency?

Reply:
Yes. There is no requirement to check for concurrency here as only the admin is capable of editing the configs and the only means of editing is through UI.

Reply:
Currently the system is designed is such that BACKEND_CONFIG can also be changed without UI, hence the concurrency check. @Jyrsa any thoughts?

Reply:
Of course, the most important thing is when there are new VARS added to BACKEND_CONFIG, it has to check for concurrency, because new vars cannot be added from the UI. This is not trivial, think of some other way to do this

Reply:
If new variables are added wouldn't it be a separate process like may be involving database migrations etc . Only place I could think there could be concurrency problem is "when the admin decide to include some new vars and at the same time he also changes some vars in the UI without refreshing (in which case he couldn't see the new vars in the UI too)" which could be very unlikely. Or is there something else I am missing here ?